src/components/tools/toolbar/ToolsDotMenu.js

```diff
@@ -160,6 +160,7 @@ export default function ToolsDotMenu({
     getSelectedTools,
     onRequestToolSelected,
     onDeleteToolSelected,
+    isAdmin,
```

Comment:
This `isAdmin` flag is extracted from the props, but it's not being used anymore?
Maybe it got lost in a rebase?